contracts/modules/PolicyController.sol

```diff
@@ -31,18 +31,19 @@ contract PolicyController is
     mapping(bytes32 => mapping(uint256 => Payout)) public payouts;
     mapping(bytes32 => uint256) public payoutCount;
 
-    bytes32[] private _processIds;
+    uint256 private _processIds;
```

Comment:
This name should be changed to `_processId` or `_lastProcessId`.

Reply:
renamed member to "_assignedProcessIds", committed on master